### PR TITLE
Branch refactor person to stall

### PR DESCRIPTION
**Pull request by teoks0199**
This will refactor persons into stalls, including method names and tests. However there are still instances of person in some files that are not changed yet. These are json files, test cases or files related to javafx.

### CODE REVIEW

**Comment by jamesebond** [comment]
lgtm

File: docs/tutorials/AddRemark.md
Comment (jamesebond) [inline docs/tutorials/AddRemark.md:1]: lgtm

**Review by jamesebond — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -172,19 +172,19 @@ Step 1. The user launches the application for the first time. The `VersionedAddr
...
 
-Step 2. The user executes `delete 5` command to delete the 5th person in the address book. The `delete` command calls `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book state.
+Step 2. The user executes `delete 5` command to delete the 5th stall in the address book. The `delete` command calls `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book state.
 
 ![UndoRedoState1](images/UndoRedoState1.png)
 
-Step 3. The user executes `add n/David …​` to add a new person. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `add n/David …​` to add a new stall. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
```
Comment (beatricecst) [inline docs/DeveloperGuide.md:179]: LGTM

File: src/main/java/seedu/address/model/stall/Email.java [hunk 2]
```diff
@@ -1,10 +1,10 @@
-package seedu.address.model.person;
+package seedu.address.model.stall;
 
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's email in the address book.
+ * Represents a Stall's email in the address book.
```
Comment (beatricecst) [inline src/main/java/seedu/address/model/stall/Email.java:7]: Should we remove the stall's email and phone

**Review by beatricecst — APPROVED** [review]

**Pull request merged**